.env.example

```
@@ -1 +1,17 @@
-DEBUG=True
+ DEBUG=True
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)